src/github/client/github-user-client.ts

```diff
@@ -58,7 +70,22 @@ export class GitHubUserClient extends GitHubClient {
 		return await this.get<Octokit.AppsListInstallationsForAuthenticatedUserResponse>("/user/installations");
 	}
 
-	private async get<T>(url, config?: AxiosRequestConfig): Promise<AxiosResponse<T>> {
-		return this.axios.get<T>(url, config);
+	public async getReference(owner: string, repo: string, branch: string): Promise<AxiosResponse<Octokit.GitGetRefResponse>> {
+		return await this.get<Octokit.GitGetRefResponse>(`/repos/{owner}/{repo}/git/refs/heads/{branch}`, {
+			urlParams: {
+				owner,
+				repo,
+				branch
+			}
+		});
 	}
+
+	public async createBranch(owner: string, repo: string, body: CreateBranchBody): Promise<AxiosResponse<Octokit.GitCreateRefResponse>> {
```

Comment:
Can we call this method `createReference` so that it should be in sync with get branch method name which is `getReference`?

Reply:
Fair, have updated.